macosx/mac-app.py

```diff
@@ -20,10 +20,10 @@
 # See https://docs.python.org/3/howto/pyporting.html for details.
 # The following code is the same used in package six to define a
```

Comment:
Please remove also lines 21 and 22 (the last two lines of the comment) since they are no longer related to the code.